src/components/Menu/Menuawaremobile.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import { withBreakpoints } from 'cozy-ui/react'
+import { ActionMenu, Menu, MenuButton, MenuItem } from 'cozy-ui/react/'
+import { UserAvatar } from '../../sharing/components/Recipient'
+import styles from 'drive/styles/actionmenu'
+
+class MenuAwareMobile extends Component {
+  state = {
+    active: false
+  }
+
+  toggle = () => {
+    this.setState({ active: !this.state.active })
+  }
+  render() {
+    const {
+      breakpoints: { isMobile },
+      children,
+      text,
+      buttonClassName,
+      name
+    } = this.props
+    if (isMobile) {
+      return (
+        <div>
+          <MenuButton
+            onClick={this.toggle}
+            text={text}
+            buttonClassName={buttonClassName}
+          />
+          {this.state.active && (
+            <ActionMenu onClose={() => this.toggle()}>
+              <MenuItem className={styles['fil-mobileactionmenu']}>
```

Comment:
Does the action menu have a title in the end?

Reply:
Nop. Only items.